client/src/pages/signUp.js

```diff
@@ -0,0 +1,179 @@
+  const [emailError, setEmailError] = useState("");
+  const [passwordError, setPasswordError] = useState("");
+  const [confirmPasswordError, setConfirmPasswordError] = useState("");
+
+  const signup = () => {
+    email.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i)
+      ? setEmailError("")
+      : setEmailError("Valid email is required!");
+    password === confirmPassword
+      ? setConfirmPasswordError("")
+      : setConfirmPasswordError("password dosen't match!");
+
+    //TO-DO
+  };
+
+  return (
+    <Grid container className={classes.root}>
+      <Grid item xs={12} sm={5} md={5} className={classes.image}>
+        <Icon className={classes.icon}>settings</Icon>
+        <Typography
+          className={classes.whiteText}
+          variant="h4"
+          color="textPrimary"
+        >
+          Welcome Back!
+        </Typography>
+        <Box width="45%">
+          <Typography variant="body" className={classes.whiteText}>
+            Already have an account? Please login to keep connections
+          </Typography>
+        </Box>
+        <Button
+          type="submit"
+          variant="contained"
+          color="primary"
+          className={classes.submit}
+        >
+          Login
+        </Button>
+      </Grid>
+      <Grid item xs={12} sm={7} md={7} component={Paper} elevation={6}>
+        <div className={classes.paper}>
+          <Typography component="h1" variant="h5">
+            Create an Account
+          </Typography>
+          <form className={classes.form}>
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              id="name"
+              label="Name"
+              name="name"
+              onChange={e => setName(e.target.value)}
+            />
+
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              id="email"
+              label="Email Address"
+              name="email"
+              autoComplete="email"
+              onChange={e => setEmail(e.target.value)}
+            />
+            {emailError}
+
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              name="password"
+              label="Password"
+              type="password"
+              id="password"
+              onChange={e => setPassword(e.target.value)}
+            />
+            {passwordError}
+
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              name="password"
+              label="Confirm Password"
+              type="password"
+              id="password"
+              onChange={e => setConfirmPassword(e.target.value)}
+            />
+            {confirmPasswordError}
```

Comment:
The error you should add as a helperText to the TextField, and you should have the error field be true if confirmPasswordError has a value